website/check_version.py

```diff
@@ -9,30 +9,15 @@
     version = f.read()
 version = version.strip()
 
-
-def is_version_up(version: str) -> bool:
-    try:
-        with warnings.catch_warnings():
-            warnings.filterwarnings("ignore")
-            r = requests.head(f"https://pypi.org/project/gradio/{version}/", timeout=3, verify=False)
-        if r.status_code == 200:
-            return True
-    except (ConnectionError, requests.exceptions.ConnectionError):
-        return False
-
 def wait_for_version(version: str):
     for _ in range(10):
-        if is_version_up(version):
+        latest_gradio_stable = requests.get("https://pypi.org/pypi/gradio/json").json()["info"]["version"]
```

Comment:
nit: Maybe it's worth putting this in its own helper as I see this logic is in separate files?

Reply:
good idea, done